templates/product.backup-battery-pdp.json

```diff
@@ -76,6 +76,12 @@
           }
         },
+        "8844cfe1-aa50-4545-8891-8470166450f7": {
+          "type": "custom_liquid",
+          "settings": {
+            "custom_liquid": "{% if product.metafields.custom.price_chart != blank %}\n    <div class=\"table-price-chart center\">\n        <table>\n            <tbody>\n                <tr>\n                    <th>Retail Price<\/th>\n                    <th>Partner Price<\/th>\n                    {% for i in product.metafields.custom.price_chart.value %}\n                        <th>{{ i.price_name }} <small class=\"chart-subtitle\">{{ i.price_subtitle }}<\/small><\/th>\n                    {% endfor %}\n                <\/tr>\n                <tr>\n                    <td>\n                        <span class=\"price table--price\">{{ product.compare_at_price | money }}<\/span>\n                        \n                    <\/td>\n                    <td>\n                        <span class=\"price partner--price\">{{ product.price | money }}<\/span>\n                        <span class=\"badge badge-chart badge-sale\">save {{ product.compare_at_price | minus: product.price | times: 100 | divided_by: product.compare_at_price }}%<\/span>\n                    <\/td>\n                    {% for i in product.metafields.custom.price_chart.value %}\n                        {% assign custom_price = i.price_value | times: 100.0 %}\n                        <td>\n                            <span class=\"price table--price\">{{ custom_price | money }}<\/span>\n                            <span class=\"badge badge-chart badge-sale\">save {{ product.compare_at_price | minus: custom_price | times: 100 | divided_by: product.compare_at_price | round }}%<\/span>\n                        <\/td>\n                    {% endfor %}\n                <\/tr>\n            <\/tbody>\n        <\/table>\n    <\/div>\n{% endif %}"
+          }
```

Comment:
can we make this a section so the HTML/Liquid can be re-used in multiple templates? If we want to make changes later it's going to have to be updated everywhere the custom liquid is inserted

Reply:
@heythisisnate  added changes. rendered as snippet file instead. https://konnected.io/products/konnected-alarm-panel-pro-12-zone-wired-alarm-conversion-dealer-kit?preview_theme_id=161737408827